styles/style.css

```diff
@@ -92,13 +98,53 @@ p {
   border-left-color: var(--orange-dark);
 }
 
+.text-white {
+  color: var(--white);
+}
+
+.text-white-bold {
+  color: var(--white);
+  font-weight: bold;
+}
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+/* INSERT BUTTON STYLES HERE */
+
+.btn {
+  padding: 0.5em;
+  border-radius: 0.4em;
+  border-color: transparent;
+  font-size: 1em;
+}
+
+.donate-btn {
+  background-color: #c05326;
+  color: #ffffff;
+}
+
+.donate-btn:hover {
+  background-color: #755444;
+}
+
+.volunteer-btn {
+  background-color: #fff;
+  color: #c05326;
+}
+
+.volunteer-btn:hover {
+  background-color: #755444;
+  color: #fff;
+}
```

Comment:
Good styling of buttons - I think the hover colours are not quite what was asked for. They should be inverted, i.e. orange becomes white, white becomes orange.

Reply:
The colours of the buttons are inverted on hover